src/computer.cpp

```diff
@@ -47,13 +100,28 @@ computer &computer::operator=(const computer &rhs)
     security = rhs.security;
     name = rhs.name;
     mission_id = rhs.mission_id;
-    options = rhs.options;
-    failures = rhs.failures;
+    options.resize(rhs.options.size());
```

Comment:
Why are you not using the assignment operator of `std::vector`? Same in the copy constructor below.